features/vibration.yml

```
@@ -0,0 +1,5 @@
+name: Vibration
+description: The `navigator.vibrate()` method provides access to the vibration mechanism of the device. Vibration is a form of tactile feedback.
```

As written, I think this implies that it returns some object with more methods. Maybe something like this would do?

```suggestion
description: The `navigator.vibrate()` method makes a tactile alert, if the device is equipped with a haptic motor.
```